### PR TITLE
Add failing test that demonstrates data files cannot be included in the build output

### PR DESCRIPTION
**Pull request by Crunch09**
Hi,
this hopefully reproduces the behavior described in the issue. I'd be happy to try to patch it :grinning:

### CODE REVIEW

**Comment by parkr** [comment]
Thanks, @Crunch09! Where is the config setting? It should be `include: _data` but I don't see it. Am I missing something?

File: features/data.feature [hunk 1]
```diff
@@ -86,6 +86,24 @@ Feature: Data
...
+    Given I have a _data directory
+    And I have a "foo" file that contains "bar"
+    And I have a "_data/members.json" file with content:
+      """
+      [{"name": "Jack", "age": 28},{"name": "Leon", "age": 34}]
+      """
+    And I have a configuration file with "include" set to:
+      | value               |
+      | foo                 |
+      | _data/members.json  |
```
Comment (pathawks) [inline features/data.feature]: @parker Setting config `include: ["value", "foo", "_data/members.json"]`

**Comment by pathawks** [comment]
Is issue is that you need to include `_data` itself rather than a specific file like `_data/members.json`?

**Comment by parkr** [comment]
> Is issue is that you need to include _data itself rather than a specific file like _data/members.json?

Interesting! If you run this test with just `_data` (and all the other values), is it successful?

**Comment by Crunch09** [comment]
@parkr Hi, yes, if understood the author of the issue correctly he wanted to include only specific files from the `_data` folder. If i pass `_data` to `includes` the test is passing as expected.

**Comment by parkr** [comment]
I believe this is acceptable as-is. You can't include anything in any directory which starts with a special character `.`, `_`, or `#`. That's mostly by design. In terms of `_data`, we offer a `data_source` configuration option which allows you to publish this data _and_ have it available for use in `site.data`. Use something like `data_source: data` (no underscore) and you'll see your data output properly in your site.

**Pull request closed without merging**